<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Comprador</class>
 <widget class="QMainWindow" name="Comprador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comprador</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget {
border-image: url(:/sgcimage/Beachfront_(image_by_Ray_Bilcliff).jpg); }


</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="11" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="7">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="0">
           <widget class="QLabel" name="LPrecoOutras">
            <property name="styleSheet">
             <string notr="true">color:white;
</string>
            </property>
            <property name="text">
             <string>Preco:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="EntradaDescricaoOutrasCompras"/>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox">
            <property name="styleSheet">
             <string notr="true">color:orange;
font: 75 10pt &quot;Noto Sans&quot;;
</string>
            </property>
            <property name="text">
             <string>Outras Compras?</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="LDescricaoOutras">
            <property name="styleSheet">
             <string notr="true">color:white;
</string>
            </property>
            <property name="text">
             <string>Descricao:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="entradaQuantidadeOutrasCompras"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="LQuantidadeOutras">
            <property name="styleSheet">
             <string notr="true">color:white;
</string>
            </property>
            <property name="text">
             <string>Quantidade:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="entradaPrecoOutrasCompras"/>
          </item>
         </layout>
        </item>
        <item row="11" column="4">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">border-color: rgb(85, 170, 255);
background-color: rgb(85, 85, 255);
color: black;
font: 87 12pt &quot;Noto Sans ExtCond Blk&quot;;</string>
            </property>
            <property name="text">
             <string>Efectuar Compra</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="4">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="6">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="4">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="7">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="4">
         <spacer name="verticalSpacer_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="11" column="7">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="4">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="12" column="4">
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="5">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="4">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="9" column="4">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="4">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="1">
           <widget class="QLabel" name="LTrocosPao">
            <property name="styleSheet">
             <string notr="true">color:white;
font: 75 11pt &quot;Noto Sans&quot;;
</string>
            </property>
            <property name="text">
             <string>00</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="entradaValorInicial"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">color:white;
</string>
            </property>
            <property name="text">
             <string>Quantidade:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="QuantidadePao"/>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="LValorTotalCompra">
            <property name="styleSheet">
             <string notr="true">color:black;
font: 75 11pt &quot;Noto Sans&quot;;
</string>
            </property>
            <property name="text">
             <string>00</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">background-color: red;
font: 75 10pt &quot;Noto Sans&quot;;
border-color: rgb(255, 255, 0);
color:white;</string>
            </property>
            <property name="text">
             <string>Valor Inicial:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">background-color: red;
border-color: rgb(255, 85, 127);
font: 75 10pt &quot;Noto Sans&quot;;
</string>
            </property>
            <property name="text">
             <string>Preco:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_3">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 0, 127);
color: white;
font: 63 12pt &quot;Noto Sans Cond SemBd&quot;;</string>
            </property>
            <property name="text">
             <string>Confirmar Compra</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="PrecoPao"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">color:white;
</string>
            </property>
            <property name="text">
             <string>Valor Total Compra:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="styleSheet">
             <string notr="true">color:white;
</string>
            </property>
            <property name="text">
             <string>Trocos:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0" colspan="8">
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="2">
           <widget class="QDateEdit" name="dateEdit"/>
          </item>
          <item row="2" column="5">
           <widget class="QLabel" name="SaldoInicial">
            <property name="styleSheet">
             <string notr="true">color:red;</string>
            </property>
            <property name="text">
             <string>00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="styleSheet">
             <string notr="true">color:white;
</string>
            </property>
            <property name="text">
             <string>Hora da Compra:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QTimeEdit" name="EHoras"/>
          </item>
          <item row="3" column="4">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">color:white;</string>
            </property>
            <property name="text">
             <string>SALDO ACTUAL:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">color:white;</string>
            </property>
            <property name="text">
             <string>SALDO INICIAL:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QLabel" name="LSaldoActual">
            <property name="styleSheet">
             <string notr="true">color:orange;</string>
            </property>
            <property name="text">
             <string>00</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_13">
            <property name="styleSheet">
             <string notr="true">color:white;
</string>
            </property>
            <property name="text">
             <string>Data da Compra:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="3">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
